.github/workflows/regression-report.yml

```diff
@@ -81,22 +86,21 @@ jobs:
         env:
           COMMENT_BODY: ${{ github.event.comment.body }}
         run: |
-          CONFIG_BUNDLE_PATH=$(echo "$COMMENT_BODY" | \
+          if [[ "$COMMENT_BODY" == "Github, generate report for configs in PR description" ]]; then
+            ./.ci/append-to-github-output.sh "mode" "pr_description"
+          elif [[ "$COMMENT_BODY" == "Github, generate report by config from"* ]]; then
+            ./.ci/append-to-github-output.sh "mode" "generate_config"
+            INPUT_FILE=$(echo "$COMMENT_BODY" | sed -E 's/.*config from //g')
+            ./.ci/append-to-github-output.sh "input_file" "$INPUT_FILE"
+          elif [[ "$COMMENT_BODY" == "Github, generate report for "* ]]; then
```

Comment:
`(Github|GitHub),`
let match by regexp - https://stackoverflow.com/a/17421041

Reply:
and move  `elif [[ "$COMMENT_BODY" == "Github, generate report by config from"* ]]; then` to bottom or top if IF-ELSE strucuture to keep `Github, generate report for` conditions being grouped together.

Reply:
> https://stackoverflow.com/a/17421041

I had tried this, but the job fails. Will try again and update.

Reply:
Done.